docs/src/components/HeaderNavDropdown.tsx

```diff
@@ -0,0 +1,78 @@
+import * as React from 'react';
+import NextLink from 'next/link';
+import { styled } from '@material-ui/core/styles';
+import Box from '@material-ui/core/Box';
+import Collapse from '@material-ui/core/Collapse';
+import ClickAwayListener from '@material-ui/core/ClickAwayListener';
+import IconButton from '@material-ui/core/IconButton';
+import SvgHambugerMenu from 'docs/src/icons/SvgHamburgerMenu';
+
+const Anchor = styled('a')(({ theme }) => ({
+  ...theme.typography.body2,
+  fontWeight: 'bold',
+  textDecoration: 'none',
+  color: theme.palette.text.secondary,
+  display: 'block',
+  padding: theme.spacing(1),
+  borderRadius: theme.spacing(1),
+  '&:hover, &:focus': {
+    backgroundColor: theme.palette.grey[200],
+  },
+}));
+
+const HeaderNavDropdown = () => {
```

Comment:
We'll probably have to think a little more about the mobile menu. We need to figure out a cool way to open the Products popper on mobile.